src/libServer/LookupServer.h

```diff
@@ -411,7 +411,7 @@ class LookupServer : public Server,
 
     auto balanceStr = resp.asString();
 
-    resp = balanceStr;
+    resp = "0x" + balanceStr;
```

Comment:
maybe while we are it, we can simplify it as
```
response = "0x" + this->GetBalance(address, true)["balance"].asString();
```

Reply:
Also, `(void)request;` is not needed anymore.